### DISCUSSION TITLE
app bridge 3.0

### DISCUSSION THREAD

**Discussion by litalboa** [Kyon147/laravel-shopify#12] [General]
Is this repo will ensure compatibility with the new Shopify admin domain?

https://shopify.dev/apps/tools/app-bridge/getting-started/app-setup#initialize-shopify-app-bridge-in-your-app

**Comment by Kyon147** [comment]
Hi @litalboa 

The `host` has been added but I do see that the `ShopOrigin` is being passed - I need to create a PR to remove that and then this should be compatible with the new admin domain.

I have tested this package on one of my live apps, that works fine on the admin url, but it is on SPA mode - so I am not sure the compatibility for blade at this moment.
